src/main/java/com/google/api/codegen/DiscoveryContext.java

```diff
@@ -240,6 +240,12 @@ public boolean hasMediaUpload(Method method) {
   }
 
   public boolean hasMediaDownload(Method method) {
+    // ignore media download for methods supporting media upload, as Apiary cannot combine both in
```

Comment:
Include a pointer to the external doc stating this:
https://developers.google.com/discovery/v1/using#discovery-doc-methods